hpc/nways/nways_labs/nways_MD/English/C/jupyter_notebook/openmp/nways_openmp_opt.ipynb

```diff
@@ -30,7 +30,7 @@
     "\n",
```

Comment:
Same comment as OpenACC. Here are some observations. By applying all techniques there is no improvement in performance.  We are using some terminologies which is difficult to understand in Nsight compute which are not covered in GPU Terminologies. We can split this into 2 jupyter like OpenACC.